src/main/java/org/testng/xml/ResultContentHandler.java

```diff
@@ -1,152 +0,0 @@
-package org.testng.xml;
-
-import static org.testng.reporters.XMLReporterConfig.ATTR_DESC;
-import static org.testng.reporters.XMLReporterConfig.ATTR_DURATION_MS;
-import static org.testng.reporters.XMLReporterConfig.ATTR_NAME;
-import static org.testng.reporters.XMLReporterConfig.ATTR_STATUS;
-import static org.testng.reporters.XMLReporterConfig.TAG_CLASS;
-import static org.testng.reporters.XMLReporterConfig.TAG_PARAMS;
-import static org.testng.reporters.XMLReporterConfig.TAG_SUITE;
-import static org.testng.reporters.XMLReporterConfig.TAG_TEST;
-import static org.testng.reporters.XMLReporterConfig.TAG_TEST_METHOD;
-
-import org.testng.ITestResult;
-import org.testng.collections.Lists;
-import org.testng.remote.strprotocol.GenericMessage;
-import org.testng.remote.strprotocol.IRemoteSuiteListener;
-import org.testng.remote.strprotocol.IRemoteTestListener;
-import org.testng.remote.strprotocol.MessageHelper;
-import org.testng.remote.strprotocol.SuiteMessage;
-import org.testng.remote.strprotocol.TestMessage;
-import org.testng.remote.strprotocol.TestResultMessage;
-import org.testng.reporters.XMLReporterConfig;
-import org.xml.sax.Attributes;
-import org.xml.sax.helpers.DefaultHandler;
-
-import java.util.List;
-
-/**
- * Parses testng-result.xml, create TestResultMessages and send them back to the listener
- * as we encounter them.
- *
- * @author Cedric Beust <cedric@beust.com>
- */
-public class ResultContentHandler extends DefaultHandler {
```

Comment:
I don't find it on https://github.com/testng-team/testng-remote/tree/rm_distributed_classes